app/views/orders/edit.slim

```diff
@@ -0,0 +1,3 @@
+| edit
+
+= render 'orders/form'
```

Comment:
https://github.com/DenisUA/Ruby-hotel/pull/15/commits/3a72381d9240a6987a893cae6a06fcf10c56f0cd 
I've already fix this